content/applications/finance/payment_acquirers/ogone.rst

```diff
@@ -5,84 +5,105 @@ Ogone
-generate a new password from :menuselection:`Configuration --> Users --> Your chosen user --> change
-password`.
+- :guilabel:`UserID`: you can choose anything you want, but the convention is to name it `OOAPI`.
+- :guilabel:`User's Name, E-mail and Timezone`: you can enter the information you want.
+- :guilabel:`Profile`: should be set to :guilabel:`Admin`.
+- :guilabel:`Special user for API`: should be checked.
 
-If you don't have a user, create one by going to :menuselection:`Configuration --> Users -->
-New User`. Set your **User ID** to get your **password** when you save your new user.
+After the creation of the user, you are required to generate a password. Save the password and
+**UserID**, as they will be required later on during the setup.
 
-.. image:: ogone/ogone_new_user.png
-   :align: center
-   :alt: Get your password when you save the new user.
+Set up Ogone for Odoo
+~~~~~~~~~~~~~~~~~~~~~
 
-.. _ogone/sha_key_in:
+Ogone must now be configured to accept payments from Odoo. Head to :menuselection:`Configuration -->
+Technical Information --> Global Security Parameters`, select :guilabel:`SHA-1` as
+:guilabel:`Hash Algorithm` and :guilabel:`UTF-8` as :guilabel:`character encoding`. Then, go to the
+:guilabel:`Data and Origin Information` tab of the same page and leave the URL field of the
+:guilabel:`e-Commerce and Alias Gateway` section blank.
 
-SHA Key IN
-~~~~~~~~~~
-
-In order to retrieve the SHA Key IN, log into your ogone account, go to
-:menuselection:`Configuration --> Technical Information --> Data and origin verification -->
-Checks for e-Commerce & Alias Gateway`, and retrieve **SHA Key IN**.
-
-.. _ogone/sha_key_out:
-
-SHA Key OUT
-~~~~~~~~~~~
-
-In order to retrieve the SHA Key OUT, log into your ogone account, go to
-:menuselection:`Configuration --> Technical Information --> Transaction feedback --> All transaction
-submission modes`, and get or generate your **API Key** and **Client Key**. Be careful to copy your
-API key as you'll not be allowed to get it later without generating a new one.
+You are now required to generate a **SHA-IN** passphrase. **SHA-IN** and **SHA-OUT** passphrases are
+used to digitally sign the transaction requests and responses between Odoo and Ogone. By using these
+secret passphrases and the **SHA-1** algorithm, both systems can ensure that the information they
+receive from the other was not altered or tampered with.
 
 .. important::
-   If you are trying Ogone as a test, with the Test Account, change the **State** to *Test Mode*. We
-   recommend doing this on a test Odoo database, rather than on your main database.
+   If you need to use another algorithm, such as **SHA-256** or **SHA-512**, within Odoo, activate
+   the :ref:`developer mode <developer-mode>` and go to :menuselection:`General Settings -->
+   Technical --> System Parameters`. From here, search for `payment_ogone.hash.function` and
+   change the value line to the desired algorithm (`SHA-256` or `SHA-512`).
```

Comment:
The values should in fact be `sha1`, `sha256`, or `sha512` (lower-case ; all attached). See the [data for the related System Parameter](https://github.com/odoo/odoo/blob/15.0/addons/payment_ogone/data/payment_acquirer_data.xml#L22) for 'proof'.